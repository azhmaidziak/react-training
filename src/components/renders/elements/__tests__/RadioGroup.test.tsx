describe("Radio Element Test: TODO IT", () => {});