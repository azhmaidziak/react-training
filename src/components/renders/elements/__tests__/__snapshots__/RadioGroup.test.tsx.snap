// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Radio Group Test Test render correctly 1`] = `
<div
  className="className"
>
  <input
    id="id_1"
    name="name_1"
    onChange={[MockFunction]}
    type="radio"
    value={0}
  />
  <label
    htmlFor="id_1"
  >
    label_1
  </label>
  <input
    id="id_2"
    name="name_2"
    onChange={[MockFunction]}
    type="radio"
    value={1}
  />
  <label
    htmlFor="id_2"
  >
    label_2
  </label>
</div>
`;
